zetaclient/orchestrator/orchestrator.go

```diff
@@ -289,122 +296,114 @@ func (oc *Orchestrator) runScheduler(ctx context.Context) error {
 		return err
 	}
 
-	observeTicker := time.NewTicker(3 * time.Second)
```

Comment:
This ticker was running every 3s. Now, it runs every block or slower. What would happen if the next tick occurred at N+2 instead of N+1?

Reply:
I'm not exactly sure what you're saying. We weren't doing anything until we got a new block.

Reply:
Is it possible to "lose" a "tick" and jump from the 100th to the 102nd block?

Reply:
Yes that is possible if zetacored restarts. But that would have resulted in a missed tick in the polling style too.

The actual work to do comes from the `GetPendingCctxsWithinRateLimit` call.

Reply:
Is missing a block a problem? 
Since we have the block number and the last block number, couldn't we use that to verify that no blocks are missed?

Reply:
No missing a block should be fine. The one scenario we should consider guarding against is if the zetacored node is resyncing. But that would break a lot of other stuff in zetaclient anyway.